qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java

```diff
@@ -58,7 +59,9 @@ public void passesCorrectFilesWithNoExceptions() throws Exception {
     /**
      * FindbugsValidator can report incorrectly added throws.
      * @throws Exception If something wrong happens inside
+     * @todo #706 Temporary disable BED_BOGUS_EXCEPTION_DECLARATION check
```

Comment:
@mkordas Could you help me, how puzzle should be described, because seems I am confused a bit, I have found this link http://www.yegor256.com/2009/03/04/pdd.html. Thanks